ACadSharp/ACadSharp.csproj

```diff
@@ -32,4 +32,8 @@
     </None>
   </ItemGroup>
 
+  <ItemGroup>
+    <PackageReference Include="System.Memory" Version="4.5.5" />
```

Comment:
Is this package needed? I see that you are using `ReadOnlySpan` to return the bytes for the `GetTrueColorRgb` and `GetIndexRGB` but is there any difference if we return an array of bytes?

Reply:
This package is implicitly included on all .NET Core projects by default as it is part of core packages.  This will only be included on the .NET Framework 4.8 and ignored by all the other targets. So effectively, it is not adding a package requirement except for the 4.8 target.

In the `GetTrueColorRgb` method, I'm using the BitConverter.GetBytes(_color) to convert the int32 into four bytes representing the individual RGB color values.  The fourth byte is used by the struct to determine if the color is a true color or indexed color.  For the end-user the fourth byte is an implementation detail they don't and shouldn't care about.  Using a ReadOnlySpan<byte> to slice this array to three bytes prevents a new allocation and copy of a new 3 byte array from the 4 byte array.

In the `GetIndexRGB` method, I'm calling the .AsSpan(); method on the `_indexRgb` byte array which allows for a reference to the contents but prevents someone from modifying the values them (by accident or otherwise).

The `System.Memory` package is also required by PR #94 extensively which allows for a zero copy reading and writing of MText Values.  I do not see this as adding anything too heavy to ACadSharp.  Trust me, I'm the first one to try to remove packages and reduce package bloat, but this is a very small concession to allow 4.8 users to use Spans for increased efficiency.

Reply:
Thanks for the clarification! I see that we are on the same page about packages.